<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teacher</class>
 <widget class="QWidget" name="Teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1063</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1063</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#ptn_last
{
	image: url(:/images/left_arrow.png);
}
QPushButton#ptn_next
{
	image: url(:/images/right_arrow.png);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>505</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="label_total">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="ptn_home">
        <property name="text">
         <string>首页</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="ptn_last">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_now">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ptn_next">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ptn_end">
        <property name="text">
         <string>尾页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ptn_NotCorrected">
        <property name="text">
         <string>未批改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ptn_released">
        <property name="text">
         <string>已销假</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ptn_agree">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>340</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptn_refuse">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>340</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>460</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: #BDC8E2;
    /*background-color: #2E3648;*/
	background-color : transparent;
	font-weight: bold;
	/*color: #BDC8E2;*/
	color: #2c3e50;

	padding-left: 10px;
	padding-top: 8px;
	padding-right: 7px;
	padding-bottom: 9px;
	
	border-style: solid;
    border-width: 2px;
    border-color: aqua;
    border-radius: 20px;

}

QLabel:hover{
	color: #2c3e50;
	border-color: #95a5a6;
    background-color: #ecf0f1;
}

QScrollArea#scrollArea
{
	background-color : transparent;
}</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>460</width>
      <height>180</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>180</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>180</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>458</width>
       <height>178</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_reasonContent">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>170</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>440</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: #BDC8E2;
    /*background-color: #2E3648;*/
	background-color : transparent;
	font-weight: bold;
	/*color: #BDC8E2;*/
	color: #2c3e50;

	padding-left: 10px;
	padding-top: 8px;
	padding-right: 7px;
	padding-bottom: 9px;
	
	border-style: solid;
    border-width: 2px;
    border-color: aqua;
    border-radius: 20px;	

}

QLabel:hover{
	color: #2c3e50;
	border-color: #95a5a6;
    background-color: #ecf0f1;
}
</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_timeBegin">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_timeEnd">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_isSchool">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: #BDC8E2;
    /*background-color: #2E3648;*/
	background-color : transparent;
	font-weight: bold;
	/*color: #BDC8E2;*/
	color: #2c3e50;

	padding-left: 10px;
	padding-top: 8px;
	padding-right: 7px;
	padding-bottom: 9px;
	
	border-style: solid;
    border-width: 2px;
    border-color: aqua;
    border-radius: 20px;	

}

QLabel:hover{
	color: #2c3e50;
	border-color: #95a5a6;
    background-color: #ecf0f1;
}</string>
   </property>
   <widget class="QLabel" name="label_leaveType">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>11</y>
      <width>109</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>请假类型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_typeContent">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>331</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: #BDC8E2;
    /*background-color: #2E3648;*/
	background-color : transparent;
	font-weight: bold;
	/*color: #BDC8E2;*/
	color: #2c3e50;

	padding-left: 10px;
	padding-top: 8px;
	padding-right: 7px;
	padding-bottom: 9px;
	
	border-style: solid;
    border-width: 2px;
    border-color: aqua;
    border-radius: 20px;	

}

QLabel:hover{
	color: #2c3e50;
	border-color: #95a5a6;
    background-color: #ecf0f1;
}</string>
   </property>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_class">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>121</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>231</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: transparent;
	font: 12px;
	font-weight: bold;
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Corrected">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/Corrected.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_CorrectedCount">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_NotCorrected">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/NotCorrected.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_NotCorrectedCount">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>widget_5</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>ptn_agree</zorder>
  <zorder>ptn_refuse</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_4</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
